src/Entity/Fileset.php

```diff
@@ -64,4 +66,26 @@ public function removeFile(File $file)
     {
         $this->files->removeElement($file);
     }
+
+    /**
+     * Check if the fileset is done processesing.
+     *
+     * @return boolean
+     */
+    public function isDone() :bool
+    {
+        $criteria = Criteria::create()
+        ->where(
+            new Comparison(
+                'status',
+                Comparison::IN,
+                array(
+                    File::FILE_NEW,
+                    File::FILE_IN_PROGRESS
```

Comment:
What about error? should we add that too?

Reply:
I think error means processed. We can add it later if FILE_ERROR has meaning. How does that sound?